test/parallel/test-vm-basic.js

```diff
@@ -128,3 +128,23 @@ const vm = require('vm');
              'Received type object'
   });
 });
+
+// vm.getWrappedFunction
+{
+  assert.strictEqual(
+    vm.getWrappedFunction('console.log("Hello, World!");'),
```

Comment:
Can we have a `.toString()` here to be explicit about what we are comparing here?

Reply:
definitely. The tests haven't been updated to reflect the latest changes made in the code.